lib/omedis/accounts/organisation/changes/create_organisation_defaults.ex

```diff
@@ -172,24 +184,38 @@ defmodule Omedis.Accounts.Organisation.Changes.CreateOrganisationDefaults do
         },
         actor: actor,
         tenant: organisation,
-        authorize?: false
+        authorize?: false,
+        upsert?: true,
+        upsert_identity: :unique_name
       )
 
     project
   end
 
   defp create_activity(project, users_group, opts) do
-    {:ok, _} =
-      Accounts.Activity.create(
-        %{
-          name: "Miscellaneous",
-          slug: "miscellaneous",
-          group_id: users_group.id,
-          project_id: project.id,
-          is_default: true,
-          color_code: "#808080"
-        },
-        opts
-      )
+    activity = fetch_default_activity(opts)
+
+    if activity do
+      activity
+    else
+      {:ok, _} =
+        Accounts.Activity.create(
+          %{
+            name: "Miscellaneous",
+            slug: "miscellaneous",
+            group_id: users_group.id,
+            project_id: project.id,
+            is_default: true,
+            color_code: "#808080"
+          },
+          opts
+        )
+    end
```

Comment:
Usually, I use `case` for checks like this one.
```suggestion
    case fetch_default_activity(opts) do
      %Accounts.Activity{} = activity ->
        activity

      nil ->
        {:ok, _} =
          Accounts.Activity.create(
            %{
              name: "Miscellaneous",
              slug: "miscellaneous",
              group_id: users_group.id,
              project_id: project.id,
              is_default: true,
              color_code: "#808080"
            },
            opts
          )
    end
```

but in this case we don't even care about the returned value, so maybe this can work:
```suggestion
    if !fetch_default_activity(opts) do
      {:ok, _} =
        Accounts.Activity.create(
          %{
            name: "Miscellaneous",
            slug: "miscellaneous",
            group_id: users_group.id,
            project_id: project.id,
            is_default: true,
            color_code: "#808080"
          },
          opts
        )
    end
```